src/main/java/org/tikv/common/util/ConcreteBackOffer.java

```diff
@@ -127,17 +127,17 @@ private BackOffFunction createBackOffFunc(BackOffFunction.BackOffFuncType funcTy
         backOffFunction =
             BackOffFunction.create(
                 TiConfiguration.getInt(TIKV_BO_REGION_MISS_BASE_IN_MS),
-                500,
+                100,
```

Comment:
what's the value in go-client？

Reply:
why changing this value?

Reply:
BoRegionMiss              = NewConfig("regionMiss", &metrics.BackoffHistogramRegionMiss, NewBackoffFnCfg(2, 500, NoJitter), tikverr.ErrRegionUnavailable)